devel/query.lua

```diff
@@ -570,28 +570,6 @@ function is_exceeding_maxlength(index)
 end
 
 --Section: table helpers
-function safe_pairs(t, keys_only)
```

Comment:
Nothing was using the `keys_only=true` path here, and it looks like it would only work for DF types anyway. It's possible for callers to ignore values with `for k in safe_pairs(t)` in any case.

Reply:
It may have had a purpose at some point, years ago.. but I don't recall what it is/was. So, `safe_pairs` is now available without defining it?

Reply:
Recently added: https://github.com/DFHack/dfhack/blob/10db894d99daf71d297f4e877b24ee77ffd37b5d/library/lua/dfhack.lua#L176

Reply:
Yep - and this script was one of the callers I had in mind. This way we won't need to worry about multiple implementations floating around.

Also, to clarify, `pairs()` for DF types returns fields in the same order as `_index_table`, which is why it (or `safe_pairs()`) would be able to replicate the old `keys_only=true` behavior.